.kiro/steering/product.md

```
@@ -0,0 +1,27 @@
+# Product Steering - Protocol: Silent Night
+
+## Product Vision
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)